.storybook/preview.js

```diff
@@ -1,5 +1,5 @@
-import IronFishUIProvider from 'components/IronFishUIProvider'
-import ColorModeSwitcher from 'components/ColorModeSwitcher'
+import IronFishUIProvider from '../components/IronFishUIProvider'
```

Comment:
I'd like to keep absolute links.

Reply:
By default, Typescript does not resolve path aliases when compiling code.
This is solved by adding an additional package and a post-compilation step.